build.gradle

```diff
@@ -0,0 +1,206 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+  }
+  dependencies {
+    classpath 'io.spring.gradle:dependency-management-plugin:1.0.0.RELEASE'
+    classpath "com.netflix.nebula:gradle-extra-configurations-plugin:3.1.0"
+    classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.4'
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.0'
+    classpath 'com.moowork.gradle:gradle-node-plugin:1.1.1'
+    classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.0'
+  }
+}
+
+// TODO improve
+def buildNumber = System.env.'TRAVIS_BUILD_NUMBER' ?: (/* SonarSource infrastructure: */ System.env.'CI_BUILD_NUMBER' ?: null)
+def versionSuffix = buildNumber == null
+  ? '-SNAPSHOT'
+  : ('.0.' + buildNumber)
+
+allprojects {
+  apply plugin: 'com.jfrog.artifactory'
+
+  group = 'org.sonarsource.sonarqube'
+  version = '6.4' + versionSuffix
+
+  repositories {
+    mavenLocal()
+    mavenCentral()
```

Comment:
I suggest to replace `mavenCentral()` by 
```
maven {
  url "https://repox.sonarsource.com/sonarsource"
}
``` 

This superset of Maven Central contains additional SonarSource builds and network access is more appropriate from SonarSource offices.